x-pack/plugins/observability/public/components/burn_rate_rule_editor/constants.ts

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  ALERT_ACTION,
+  HIGH_PRIORITY_ACTION,
+  LOW_PRIORITY_ACTION,
+  MEDIUM_PRIORITY_ACTION,
+} from '../../../common/constants';
+
+import { WindowSchema } from '../../typings';
+
+type PartialWindowSchema = Partial<WindowSchema>;
+
+const WEEKLY: PartialWindowSchema[] = [
+  {
+    burnRateThreshold: 3.36,
+    longWindow: { value: 1, unit: 'h' },
+    shortWindow: { value: 5, unit: 'm' },
```

Comment:
What is short window used for? As far as I can tell, the long window is what is used for the default burn rate windows in the rule definition

Reply:
The short windows is 1/12 of the long window, when both windows are above the threshold the alert will trigger. This helps reduce the noise of short bursts of error triggering alerts, you end up with alerts for sustained events. I think we could have calculated that dynamically but I think at the time when we originally wrote this rule we weren't sure if that should be configurable by the user so it's in the API.